templates/ai/services.html

```diff
@@ -7,316 +7,306 @@
-          height="191",
-          width="335",
-          hi_def=True,
-          loading="lazy",
+    <div class="col-4 u-align--center u-hide--small">
+      {{ image (
+        url="https://assets.ubuntu.com/v1/f887a7db-deploy+fast.svg",
+        alt="",
+        width="263",
+        height="150",
+        hi_def=True,
+        loading="lazy"
         ) | safe
       }}
     </div>
   </div>
+</section>
 
-  <div class="u-fixed-width">
-    <hr class="p-separator"/>
-  </div>
-
-  <div class="row">
-    <div class="col-6 u-hide--small u-align--center">
-      {{
-        image(
-          url="https://assets.ubuntu.com/v1/455740b4-3+design_AW.svg",
-          alt="",
-          height="191",
-          width="335",
-          hi_def=True,
-          loading="lazy",
+<section class="p-strip--light">
+  <div class="row u-vertically-center">
+    <div class="col-4 u-align--center u-hide--small">
+      {{ image (
+        url="https://assets.ubuntu.com/v1/a22613fc-train-with-ubuntu_AW.svg",
+        alt="",
+        width="190",
+        height="190",
+        hi_def=True,
+        loading="lazy"
         ) | safe
       }}
+      <div>
+        <p style="margin-top: -1rem;" class="p-muted-heading">IN PARTNERSHIP WITH:</p>
+          {{ image (
+            url="https://assets.ubuntu.com/v1/4566e59b-mavencode-logo.png",
+            alt="Mavencode",
+            width="155",
+            height="19",
+            hi_def=True,
+            loading="lazy"
+            ) | safe
+          }}
+      </div>
+      <p>
+        <a class="p-link--external" href="https://canonical.com/partners/become-a-partner"><small>Become a trusted partner</small></a>
+      </p>
     </div>
-    <div class="col-6">
-      <h4>3 - Design</h4>
+    <div class="col-8">
+      <h2>Enterprise MLOps training</h2>
+      <p class="p-heading--4">4&ndash;days to success with machine learning on Kubeflow</p>
+      <p>Grow end-to-end machine learning knowledge within your team, with enterprise training by Canonical.</p>
+      <p>Align your sysadmins, devops, data engineers and data scientists with the latest MLOps solutions.</p>
+      <p>Any infra and level of expertise. Tailored to your use cases. Up to 15 attendees. 4-day on-premise training for $29.500.</p>
       <p>
-        Prototype new solutions and identify a candidate solution that seems to be the best fit for the use case under consideration. Data engineers will prepare the data, data scientists will discuss and select appropriate features and machine learning algorithms, and machine Learning engineers will design, build and perform preliminary tests on your prototype neural network. Time permitting, the iterative process of design and discovery on the data and the neural network model will start.
+        <a class="p-link--external p-button--neutral" href="https://assets.ubuntu.com/v1/ee5a96f3-Kubeflow+Training+Curriculum.pdf?_ga=2.202291033.951646160.1607941648-154096706.1586788067">See Curriculum</a>
```

Comment:
```suggestion
        <a class="p-button--neutral" href="https://assets.ubuntu.com/v1/ee5a96f3-Kubeflow+Training+Curriculum.pdf?_ga=2.202291033.951646160.1607941648-154096706.1586788067">See the curriculum</a>
```

We don't do external links for assets.u.c